packages/twenty-front/src/modules/settings/admin-panel/components/SettingsAdminEnvVariables.tsx

```diff
@@ -1,70 +1,75 @@
-import { useState } from 'react';
-import { Button, H1Title, H1TitleFontColor, H2Title, Section } from 'twenty-ui';
+import { t } from '@lingui/core/macro';
+
+import {
+  Banner,
+  Card,
+  H2Title,
+  IconHeartRateMonitor,
+  IconInfoCircle,
+  Section,
+} from 'twenty-ui';
 import { useGetEnvironmentVariablesGroupedQuery } from '~/generated/graphql';
+import { getSettingsPath } from '~/utils/navigation/getSettingsPath';
 
 const StyledGroupContainer = styled.div`
   margin-bottom: ${({ theme }) => theme.spacing(6)};
 `;
 
-const StyledGroupDescription = styled.div`
-  margin-bottom: ${({ theme }) => theme.spacing(4)};
+const StyledBanner = styled(Banner)`
+  align-items: center;
+  background-color: ${({ theme }) => theme.color.blueAccent20};
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(2)};
+  justify-content: space-between;
 `;
 
-const StyledButtonsRow = styled.div`
+const StyledBannerContent = styled.div`
+  align-items: center;
   display: flex;
-  flex-wrap: wrap;
   gap: ${({ theme }) => theme.spacing(2)};
-  margin-bottom: ${({ theme }) => theme.spacing(6)};
 `;
 
-const StyledShowMoreButton = styled(Button)<{ isSelected?: boolean }>`
-  ${({ isSelected, theme }) =>
-    isSelected &&
-    `
-    background-color: ${theme.background.transparent.light};
-  `}
+const StyledBannerText = styled.div`
+  color: ${({ theme }) => theme.color.blue};
 `;
 
 export const SettingsAdminEnvVariables = () => {
+  const theme = useTheme();
   const { data: environmentVariables, loading: environmentVariablesLoading } =
     useGetEnvironmentVariablesGroupedQuery({
       fetchPolicy: 'network-only',
     });
 
-  const [selectedGroup, setSelectedGroup] = useState<string | null>(null);
-
-  const toggleGroupVisibility = (groupName: string) => {
-    setSelectedGroup(selectedGroup === groupName ? null : groupName);
-  };
-
-  const hiddenGroups =
-    environmentVariables?.getEnvironmentVariablesGrouped.groups.filter(
-      (group) => group.isHiddenOnLoad,
-    ) ?? [];
-
   const visibleGroups =
     environmentVariables?.getEnvironmentVariablesGrouped.groups.filter(
       (group) => !group.isHiddenOnLoad,
     ) ?? [];
 
-  const selectedGroupData =
-    environmentVariables?.getEnvironmentVariablesGrouped.groups.find(
-      (group) => group.name === selectedGroup,
-    );
-
   if (environmentVariablesLoading) {
     return <SettingsAdminTabSkeletonLoader />;
   }
 
   return (
     <>
       <Section>
-        These are only the server values. Ensure your worker environment has the
-        same variables and values, this is required for asynchronous tasks like
-        email sync.
+        <StyledBanner>
+          <StyledBannerContent>
+            <IconInfoCircle
+              color={theme.color.blue}
+              size={theme.icon.size.md}
+            />
+            <StyledBannerText>
+              {t`Matching values required in worker environment`}
```

Comment:
As discussed, we should follow up on that as this isn't clear. Maybe work on DB level env variables next...